sdk/storage/azfile/file/constants.go

```diff
@@ -41,6 +41,28 @@ func PossibleCopyStatusTypeValues() []CopyStatusType {
 	return generated.PossibleCopyStatusTypeValues()
 }
 
+// NFSFileType specifies the type of the file or directory.
+type NFSFileType = generated.NfsFileType
+
+// ModeCopyMode specifies the mode of the file or directory.
+type ModeCopyMode = generated.ModeCopyMode
+
+// OwnerCopyMode specifies the copy mode source or override.
+type OwnerCopyMode = generated.OwnerCopyMode
+
+const (
+	// NFSFileTypeRegular Default and only value for the parameter NFS File Type.
+	NFSFileTypeRegular   NFSFileType = generated.NfsFileTypeRegular
+	NfsFileTypeDirectory NFSFileType = generated.NfsFileTypeDirectory
+	NfsFileTypeSymlink   NFSFileType = generated.NfsFileTypeSymLink
```

Comment:
Just to be in sync with the above type
```suggestion
	NFSFileTypeDirectory NFSFileType = generated.NfsFileTypeDirectory
	NFSFileTypeSymlink   NFSFileType = generated.NfsFileTypeSymLink
```

Reply:
Changed